src/main/avro/LandingZone.avsc

```diff
@@ -3,6 +3,14 @@
   "name": "LandingZoneModel",
   "namespace": "bio.terra.aws.resource.discovery.avro",
   "fields": [
+    {
```

Comment:
Would we have a single VPC for the entire landing zone to  reduce data transfer costs?
1) if yes, do we need a separate vpc for appFramework?
2) if no, does it make more sense to remove the appFramework prefix?

Reply:
My intent was to have a dedicated VPC for Michelangelo application VM's in ach LZ, thus the appFramework prefix.